dev-shells/default.nix

```diff
@@ -10,5 +10,41 @@ inputs.self.library.defaultSystems (
 
       inherit (inputs.self.checks.${system}.pre-commit-check) shellHook;
     };
+
+    logoDev = pkgs.mkShell {
+      packages = [
+        pkgs.poetry
+        (pkgs.python3.withPackages (
+          ps: with ps; [
+            coloraide
+            fontforge
+            freetype-py
+            jsonpickle
+            svg-py
+          ]
+        ))
+      ];
+    };
+
+    nixoslogo-dev = pkgs.callPackage (
+      {
+        mkShell,
+        python3,
+        route159,
+      }:
+      mkShell {
+
```

Comment:
```suggestion
```

Reply:
Nah. I like the extra white space.

Reply:
Fair enough, then why not making it consistent throughout the changes you're introducing here? Sometimes you add it, sometimes you don't, where's the consistency?

Reply:
Thanks for point that out. I haven't really been focusing on that. More on getting things working and was going to do a pass later.